roles/edpm_nova/tasks/derive_pci.yml

```diff
@@ -0,0 +1,74 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: EDPM Node derive_novapci_whitelist Config
+  hosts: "{{ edpm_override_hosts | default('all', true) }}"
+  strategy: linear
+  become: true
+  tags:
+    - nova
+  tasks:
+    - name: Check if edpm node has the sriov nova conf
```

Comment:
@vcandapp instead of hard coding filename for source filename, can we consider all the user input configmap files to identify soucrce file using pci section key?

Reply:
Done